### PR TITLE
Fix AT ST_BUFFER with invalid polygon in Bigquery

### PR DESCRIPTION
**Pull request by vdelacruzb**
# Description

Shortcut

- Story: https://app.shortcut.com/cartoteam/story/264442/at-st-buffer-with-invalid-polygon
- Autolink: AT ST BUFFER with invalid polygon [sc-264442]

## Type of change

- Fix

# Acceptance

```
SELECT `carto-un`.carto.ST_BUFFER(geom, 1, 'kilometers', 8)
FROM `carto-demo-data.demo_tables.world_airports`
where cartodb_id='44952';
-- DOESN'T WORKS

SELECT `cartodb-data-engineering-team`.vdelacruz_carto.ST_BUFFER(geom, 1, 'kilometers', 8)
FROM `carto-demo-data.demo_tables.world_airports`
where cartodb_id='44952';
-- WORKS
```

### CODE REVIEW

**Review by Jesus89 — APPROVED** [review]

File: clouds/bigquery/modules/test/transformations/ST_BUFFER.test.js [hunk 1]
```diff
@@ -46,4 +46,10 @@ test('ST_BUFFER should fail with wrong arguments', async () => {
 
     query = `SELECT \`@@BQ_DATASET@@.ST_BUFFER\`(ST_GEOGFROMGEOJSON('${featureJSON}'), 1, 'kilometers', -10);`;
     await expect(runQuery(query)).rejects.toThrow();
+});
+
+test('ST_BUFFER should not fail with geographies close to the poles', async () => {
```
Comment (Jesus89) [inline clouds/bigquery/modules/test/transformations/ST_BUFFER.test.js:51]: Does this test fail without the change?

Comment (vdelacruzb) [reply clouds/bigquery/modules/test/transformations/ST_BUFFER.test.js:51]: Yep it fails

Comment (Jesus89) [reply clouds/bigquery/modules/test/transformations/ST_BUFFER.test.js:51]: :rocket:

**Comment by vdelacruzb** [comment]
> @vdelacruzb could you merge the master branch in the PR to have only the new changes?

This PR is in sync with master. It seems that the linter didn't complain about some spaces left in the voronoi PR, but doing `make lint` make them back in.

**Comment by Jesus89** [comment]
OK, I mentioned that because the diff contains changes from other PR. But I have reviewed the new changes and LGTM

**Pull request merged**